library/ssl_tls.c

```diff
@@ -6050,7 +6050,7 @@ static int ssl_parse_certificate_verify( mbedtls_ssl_context *ssl,
         have_ca_chain = 1;
 
         MBEDTLS_SSL_DEBUG_MSG( 3, ( "use CA callback for X.509 CRT verification" ) );
-        ret = mbedtls_x509_crt_verify_with_cb(
+        ret = mbedtls_x509_crt_verify_with_ca_cb(
```

Comment:
Note for the future: it would be cleaner if this had been done in the same commit that changed the function name in its definition. We should try to make sure each commit at least builds on its own (and if possible, pass tests) unless there's a good reason not to. For example, writing tests for functions before they're implemented is a good reason for the tests not to build for a few commits. But other than this kind of choice, most commits should build cleanly on their own.

(No need to rewrite history for that, we tend to avoid rewriting history while review is in progress - just mentioning that for future PRs.)